### PR TITLE
Update Google Compute Engine docs

### PR DESCRIPTION
**Pull request by Kami**
It looks like some of the links have changed (the one previously documented doesn't take you to the correct place anymore).

This pull request updates the docs to use new and working link and also adds some screenshot for people not familiar with GCE console.

### CODE REVIEW

**Comment by pquentin** [comment]
@Kami Any idea why we no longer have codecov reports?

File: docs/compute/drivers/gce.rst [hunk 1]
```diff
@@ -68,11 +77,41 @@ the legacy P12 format.
...
+
+4. You will also need your "Project ID" (a string, not a numerical value) that
    can be found by clicking on the "Overview" link on the left sidebar.
 
+.. figure:: /_static/images/misc/gce/project_dashboard.png
+    :align: center
+    :width: 500
+
+5. You will also need to have billing information associated and enabled for
+   that project. If billing is not yet enabled for that project, error message
```
Comment (pquentin) [inline docs/compute/drivers/gce.rst]: *an* error message?

File: docs/compute/drivers/gce.rst [hunk 2]
```diff
@@ -53,9 +53,18 @@ To set up Service Account authentication, you will need to download the
...
+   new project (https://console.cloud.google.com/projectcreate) or re-use an
+   existing one.
+
+.. figure:: /_static/images/misc/gce/create_service_account.png
+    :align: center
+    :width: 500
+
+2. Select the existing or newly created project and go to IAM & Admin ->
+   Service Accounts -> Create service account to create a new service account.
+   Select "furnish a new private key" to create and download new private key you will
```
Comment (pquentin) [inline docs/compute/drivers/gce.rst]: Furnish with a capital F?

**Review by pquentin — APPROVED** [review]

**Comment by Kami** [comment]
@pquentin Not sure, need to have a look.

**Comment by Kami** [comment]
It seems like coverage reports are indeed getting uploaded to codecov, but for some reasons, coverage comments are not added to PRs.

**Pull request merged**